### DISCUSSION TITLE
spaCy Turkish models are ready

### DISCUSSION THREAD

**Discussion by DuyguA** [explosion/spaCy#12390] [Language Support]
Please find the 3 packages `tr_core_news_lg`, `tr_core_news_md` and `tr_core_news_trf` trained by myself: 
https://huggingface.co/turkish-nlp-suite

I also provided some documentation :wink: 

- Text mining using the brand  models: 
[Quick recipes with spaCy Turkish] (https://www.youtube.com/watch?v=w0WCkgCOzzw)
-  Code for the above playlist:  [spaCy TR shorts Github](https://github.com/turkish-nlp-suite/tr-spacy-shorts) 
- How I prepped the packages: [How spaCy Turkish packages are prepped](https://www.youtube.com/watch?v=dTdpRP-t920&list=PLJTHlIwB8Vcp_1b1eFwKcKKmzfs16EFtH)
- Config files for training the packages:  [Github repo](https://github.com/turkish-nlp-suite/turkish-spacy-models)

Please enjoy as you wish :blush:

**Comment by svlandeg** [comment]
Hi Duygu, good to hear from you, and thanks for sharing!

**Comment by joprice** [comment]
Could this be published for more recent spacy versions? I get the following conflict when trying with the latest spacy: 

```
And because spacy[lookups] (3.8.5) depends on spacy (3.8.5)
 and tr-core-news-trf (3.4.2) @ https://huggingface.co/turkish-nlp-suite/tr_core_news_trf/resolve/main/tr_core_news_trf-1.0-py3-none-any.whl depends on spacy (>=3.4.2,<3.5.0), tr-core-news-trf (3.4.2) @ https://huggingface.co/turkish-nlp-suite/tr_core_news_trf/resolve/main/tr_core_news_trf-1.0-py3-none-any.whl is incompatible with spacy[lookups] (>=3.8.4,<4.0.0).
```

since the model has a bound on spacy<3.5.0.

**Reply by DuyguA** [reply]
Hellos,
I had some more requests for this issue, I'll re-make the .whl file asap. I have time by the beginning of the June, will do the update ca in 2 weeks and give update from here.

**Comment by baharzafer** [comment]
What would it take to make these official spaCy models? Also, are there any third-party benchmarks or performance evaluations available?

**Comment by DuyguZipped** [comment]
Hellos, thanks for the question.
In theory only the models prepared by Explosion AI itself are the official models, however due to strong performance, proper documentation and offering a research perspective these are the official models.

Benchmark values are per model. For an example, for the md model, go to [HF page](https://huggingface.co/turkish-nlp-suite/tr_core_news_md) , navigate to the right handside to `Evaluation Results` and click to expand. If you need mored detailed evals i.e. f1 score per tag, I have them documented , you can jusk ask from here.
